src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -92,13 +92,20 @@ def generate_WDB_1_default_config(self, filename):
 
     conditionals = [conditionals_b]
 
```

Comment:
I think it makes the most sense to save the PAL records above to a file, then use that file in the PAL DB config. Otherwise it's not clear that the CBSDs and the PAL records will be self-consistent (same user ID, right license area, etc.). I suggest that you take the PAL records created above, save them to a file during the default config generation (same format as the file you already have), then give that filename as part of the default config.

The same applies to the CPI information, since you will only need the private keys when creating the config (and not during the test case itself).

Reply:
Updated as per suggestion.
Thanks for the suggestion. This is a good way of making CBSDs and PAL records consistent and use the same in the database.